contributor_docs/fes_contribution_guide.md

```diff
@@ -484,7 +484,7 @@ The functionality described under `fesErrorMonitor()` currently only works on th
 
 ### Performance Issue with the FES
 
-By default, FES is enabled for p5.js and disabled in `p5.min.js` to prevent FES functions from slowing down the process. The error-checking system can significantly slow down your code (up to \~10x in some cases). See the [Friendly Error performance test](https://github.com/processing/p5.js-website/tree/main/src/assets/learn/performance/code/friendly-error-system).
+By default, FES is enabled for p5.js and disabled in `p5.min.js` to prevent FES functions from slowing down the process. The error-checking system can significantly slow down your code (up to \~10x in some cases). See the [Friendly Error performance test](https://github.com/processing/p5.js/blob/main/contributor_docs/friendly_error_system.md).
```

Comment:
```suggestion
By default, FES is enabled for p5.js and disabled in `p5.min.js` to prevent FES functions from slowing down the process. The error-checking system can significantly slow down your code (up to \~10x in some cases).
```

Reply:
Makes sense. Thanks